maat-court-data-api/src/testSqsIntegration/resources/application.yaml

```diff
@@ -107,10 +106,6 @@ swagger-ui:
 
 feature:
   postMvpEnabled: true
-  prosecution-concluded-listener:
-    enabled: true
-  prosecution-concluded-schedule:
-    enabled: true
```

Comment:
Need to also remove this configuration in the main `application.yaml`

Reply:
Removed